local-storage/src/LocalStorage.ts

```diff
@@ -0,0 +1,97 @@
+import { CurriedFunction2, curry2 } from '@most/prelude';
+import {
+  LocalStorageClearCommand,
+  LocalStorageCommand,
+  LocalStorageRemoveItemCommand,
+  LocalStorageSetItemCommand,
+  LocalStorageSource,
+} from './types';
+import {
+  Stream,
+  constant,
+  drain,
+  filter,
+  fromEvent,
+  map,
+  merge,
+  startWith,
+} from 'most';
+
+import { hold } from '@most/hold';
+
+export interface LocalStorageSinks {
+  localStorage$: Stream<LocalStorageCommand>;
+}
+
+export interface LocalStorageSources {
+  localStorage: LocalStorageSource;
+}
+
+export function LocalStorage(sinks: LocalStorageSinks): LocalStorageSources {
```

Comment:
What benefit do we get by using `LocalStorageSinks` instead of `Stream<LocalStorageCommand>`? Are the README examples broken because they're actually returning `localStorage` from `Main` and there is no evidence about adding $ in `Effects`?